tests/dapr_tests.mk

```diff
@@ -90,19 +90,19 @@ DAPR_CONTAINER_LOG_PATH?=./dist/container_logs
 DAPR_TEST_LOG_PATH?=./dist/logs
 
 ifeq ($(DAPR_TEST_STATE_STORE),)
-DAPR_TEST_STATE_STORE=redis
+DAPR_TEST_STATE_STORE=postgres
 endif
 
 ifeq ($(DAPR_TEST_QUERY_STATE_STORE),)
-DAPR_TEST_QUERY_STATE_STORE=mongodb
+DAPR_TEST_QUERY_STATE_STORE=postgres
 endif
 
 ifeq ($(DAPR_TEST_PUBSUB),)
 DAPR_TEST_PUBSUB=redis
 endif
 
 ifeq ($(DAPR_TEST_CONFIG_STORE),)
-DAPR_TEST_CONFIG_STORE=redis
+DAPR_TEST_CONFIG_STORE=postgres
```

Comment:
Wait, we need this to be Redis. We need to test the config store with Redis, and there's another pipeline that tests with Postgres

Reply:
Done :+1: